specification.markdown

```diff
@@ -173,21 +173,65 @@ An EBML Stream is a file that consists of one or many EBML Documents that are co
 
-    Level:
-    EBML ID:    []
-    Mandatory:  [Mandatory]
-    Multiple:   [For level 0 Elements, this attribute defines whether or not the Element may be used multiple times within an EBML Document. For Elements at Level 1 or greater, this attribute defines whether or not the Element may be used multiple times within its parent Element.]
-    Range:
-    Default:    [Default value if the element is not found]
-    Element Type:
-    Description:
+As an XML Document the EBML Schema MUST use `<table>` as the top level element. The `<table>` element may contain `<element>` sub-elements. Each `<element>` node defines one EBML Element through the use of several attributes. Each attribute of the `<element>` node of the EBML Schema is defined here along with a note to say if the attribute is mandatory or not. EBML Schemas many contain additional attributes to extend the semantics but MUST not conflict is the definitions of the `<element>` attributes defined here.
+
+Within the EBML Schema each EBML Element is defined to occur at a specific level. For any specificied EBML Element that is not at level 0, the Parent EBML Element refers to the EBML Master-element that that EBML Element is contained within. For any specifiied EBML Master-element the Child EBML Element refers to the EBML Elements that may be immediately contained within that Master-element. For any EBML Element that is not defined at level 0, the Parent EBML Element may be identified by the preceding `<element>` node which has a lower value as the defined `level` attribute. The only exception for this rule are Global EBML Elements which may occur within any Parent EBML Element within the restriction of the Global EBML Element's range declaration.
+
+#### EBML Schema Element Attributes
+
+Within an EBML Schema the `<element>` uses the following attributes to define an EBML Element.
+
+| attribute name | required | definition |
+|----------------|----------|------------|
+| name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
+| level          | Yes      | The level notes at what hierarchical depth the EBML Element may occur within an EBML Document. The initial EBML Element of an EBML Document is at level 0 and the Elements that it may contain are at level 1. The level MUST be expressed as an integer; however, the integer may be followed by a '+' symbol to indicate that the EBML Element is valid at any higher level.  |
```

Comment:
Agree. I have to ask, though, what's the point of `level`? If specdata.xml is changed to use nested elements, then `level` seems like redundant information.

A separate field like `nested` could use the names of elements as its value, instead of an integer (which would be kind of nice, as you could then have elements that are legal in some but not all others).